test/test_cli.py

```diff
@@ -59,6 +60,7 @@ def test_smoke_test(run, client, tmp_path: Path):
     )
 
 
+@pytest.mark.skip(reason="CLI tests needs fixing")
```

Comment:
Let's create an issue or a task to fix these skipped tests in a later PR, just to avoid we forget

Reply:
This is a merge to a branch, not to main, and they're one of the main items before this can PR to main. Happy to have a ticket to track the work though.